### DISCUSSION TITLE
Start a new job after phase 4, but before copying to the final file

### DISCUSSION THREAD

**Discussion by sergestina** [ericaltendorf/plotman#449] [General]
Hi!
How can I start a new job immediately after the end of phase 4 without waiting for copying to the destination?
I am putting files into a slow FTP intermediate over a 100 Mbps link. I want to run jobs without waiting for a 3 hour copy to finish. Thank you.

**Comment by altendky** [comment]
How is the FTP setup as part of the plotman process?

**Reply by sergestina** [reply]
I mounted FTP server like local fs via CurlFtpFS. For plotman it looks like local fs:

```
Filesystem                                                Size  Used Avail Use% Mounted on
...
curlftpfs#ftp://user:password@11.1111.11.111/  954G     0  954G   0% /home/chia/ftp
...
```

**Reply by altendky** [reply]
There isn't anything implemented for this yet.  The general intended use is to use archiving to transfer plots to their final resting place, regardless of whether it is local or remote.  This feature may be added.
